tests/testthat/parse_comments/within_spacing_with_force-out.R

```diff
@@ -1,6 +1,6 @@
-####### A comment
-#' roxygen
+#######           A comment
+#'roxygen
```

Comment:
We don't do any normalisation of comments currently. Something to think about though.